notebooks/pointing_game_example.ipynb

```diff
@@ -0,0 +1,104 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "import PIL.Image as Image\n",
+    "\n",
+    "import torchvision.models.detection as d\n",
+    "\n",
+    "import PIL.Image as Image\n",
+    "from ml_wrappers.model.image_model_wrapper import (PytorchDRiseWrapper)\n",
+    "\n",
+    "from vision_explanation_methods.explanation_evaluation.pointing_game import PointingGame"
```

Comment:
explanation_evaluation seems quite long and is sort of duplicated by the "explanation" part of "vision_explanation_methods" top-level namespace.  What do you think about just evaluation?